sql/core/src/test/scala/org/apache/spark/sql/execution/SQLViewSuite.scala

```diff
@@ -52,6 +52,19 @@ abstract class SQLViewSuite extends QueryTest with SQLTestUtils {
     }
   }
 
+  test("column resolution scenarios with local temp view") {
+    val df = Seq(2).toDF("i1")
+    df.createOrReplaceTempView("table1")
+    withTempView("table1") {
+      checkAnswer(spark.sql("SELECT table1.* FROM table1"), Row(2))
+      checkAnswer(spark.sql("SELECT * FROM table1"), Row(2))
+      checkAnswer(spark.sql("SELECT i1 FROM table1"), Row(2))
+      checkAnswer(spark.sql("SELECT table1.i1 FROM table1"), Row(2))
+      checkAnswer(spark.sql("SELECT a.i1 FROM table1 AS a"), Row(2))
+      checkAnswer(spark.sql("SELECT i1 FROM table1 AS a"), Row(2))
```

Comment:
Hi Xiao,  I have moved my new local temp view tests and the global temp view tests to the SQLQueryTestSuite framework as well.  Please take a look. Thanks.